holoviews/operation/datashader.py

```diff
@@ -937,9 +937,14 @@ def _process(self, element, key=None):
         # Disable upsampling by clipping size and ranges
         (xstart, xend), (ystart, yend) = (x_range, y_range)
         xspan, yspan = (xend-xstart), (yend-ystart)
-        interp = self.p.interpolation or None
-        if interp == 'bilinear': interp = 'linear'
-        if not (self.p.upsample or interp is None) and self.p.target is None:
+        interp = self.p.interpolation
+        # Convert interpolation method to datashader format
+        # False/None means no interpolation (use nearest neighbor)
+        if interp in (False, None):
+            interp = 'nearest'
+        elif interp == 'bilinear':
+            interp = 'linear'
+        if not (self.p.upsample or interp in (False, None)) and self.p.target is None:
```

Comment:
```suggestion
        if not (self.p.upsample or interp == "nearest") and self.p.target is None:
```

It can never be False and None because of the if statement above.

Reply:
Yeah, true.